OnlTools/Jevp/StJevpPlot/DisplayDefs.cxx

```diff
@@ -7,7 +7,7 @@
 #include <math.h>
 
 #include "DisplayDefs.h"
-#include <libxml2/libxml/xmlreader.h>
+#include "libxml/xmlreader.h"
```

Comment:
```suggestion
#include <libxml/xmlreader.h>
```
for system headers

Reply:
Let's reserve angle brackets for standard library header files

Reply:
That is not a common convention, you also violate it in the same commit

Reply:
What is the common convention?

Reply:
The common convention is that parts of the same program use `""` and external libraries use `<>`. The I would assume that StA would have
```
#include <math.h>

#include <libxml/xmlreader.h>

#include <StB/StB.h>

#include "StA.h"
```

I believe we mostly do
```
#include "StB/StB.h"

#include "StA/StA.h"
```
which is okay too

Reply:
Fine